frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -85,14 +85,30 @@ pods:
         {{/TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
       {{/TASKCFG_ALL_SECURITY_KERBEROS_ENABLED}}
 
+          # Getting needed env variables from bootstrap
+          ./bootstrap -resolve=false 2> bootstrap_stderr
+          cat bootstrap_stderr 1>&2
+          awk '/^[0-9]+\/[0-9]+\/[0-9]+ /{if (pr) {pr=0}}{if (pr) {print}}/Bootstrapping with environment/{pr=1}' bootstrap_stderr > bootstrap_env
+          eval `grep "MESOS_CONTAINER_IP" bootstrap_env`
+          eval `grep "LIBPROCESS_IP" bootstrap_env`
+          export MESOS_CONTAINER_IP LIBPROCESS_IP
+
           # setup-helper determines the correct listeners and security.inter.broker.protocol.
           ./setup-helper
           export SETUP_HELPER_ADVERTISED_LISTENERS=`cat advertised.listeners`
           export SETUP_HELPER_LISTENERS=`cat listeners`
           export SETUP_HELPER_SECURITY_INTER_BROKER_PROTOCOL=`cat security.inter.broker.protocol`
           export SETUP_HELPER_SUPER_USERS=`cat super.users`
 
-          ./bootstrap -resolve=false
+          # Updating server.properties file with parameters obtained by setup-helper
```

Comment:
It is important to *not* replace the `./bootstrap --resolve=false` call here. Rather ensure that the environment is set up correctly so as to use the mustache templating in the files and not introduce a different templating mechanism (`sed`).

Reply:
OK.